provider/go.mod

```diff
@@ -6,12 +6,12 @@ replace (
 	github.com/hashicorp/go-getter v1.5.0 => github.com/hashicorp/go-getter v1.4.0
 	github.com/hashicorp/terraform-plugin-sdk/v2 => github.com/pulumi/terraform-plugin-sdk/v2 v2.0.0-20220725190814-23001ad6ec03
 	github.com/minio/minio => github.com/minio/minio v0.0.0-20210409155609-0ddc4f00756b
-	github.com/minio/minio-go/v7 => github.com/minio/minio-go/v7 v7.0.11-0.20210302210017-6ae69c73ce78
+	github.com/minio/minio-go/v7 => github.com/minio/minio-go/v7 v7.0.47
 	github.com/shirou/gopsutil/v3 => github.com/shirou/gopsutil/v3 v3.21.1
 )
 
 require (
-	github.com/aminueza/terraform-provider-minio v1.6.0
+	github.com/aminueza/terraform-provider-minio v1.10.0
 	github.com/pulumi/pulumi-terraform-bridge/v3 v3.28.0
```

Comment:
You need bump the bridge to the latest version (v3.38.1).

Reply:
let's include this in the upgrade guide instructions